src/main/java/com/doeiqts/pizzavoter/domain/Pizza.java

```diff
@@ -1,55 +1,45 @@
 package com.doeiqts.pizzavoter.domain;
 
-import com.doeiqts.pizzavoter.enums.Crust;
-import com.doeiqts.pizzavoter.enums.Sauce;
-import com.doeiqts.pizzavoter.enums.Size;
-import com.doeiqts.pizzavoter.enums.Topping;
+import com.doeiqts.pizzavoter.enums.*;
```

Comment:
Hmm... I need to make a checkstyle thing that will enforce the way I want things to look.